go-controller/pkg/ovn/master.go

```diff
@@ -1253,11 +1254,12 @@ func (oc *Controller) deleteNodeLogicalNetwork(nodeName string) error {
 	return nil
 }
 
-func (oc *Controller) deleteNode(nodeName string, hostSubnets []*net.IPNet) {
+func (oc *Controller) deleteNode(nodeName string, hostSubnets []*net.IPNet) error {
 	// Clean up as much as we can but don't hard error
 	for _, hostSubnet := range hostSubnets {
 		if err := oc.deleteNodeHostSubnet(nodeName, hostSubnet); err != nil {
 			klog.Errorf("Error deleting node %s HostSubnet %v: %v", nodeName, hostSubnet, err)
```

Comment:
we dont need to klog these now that you changed it to return an error. We can just return the error and log it from the caller.